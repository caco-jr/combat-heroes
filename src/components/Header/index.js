import React from 'react';

const Header = () => {
  return <header>Combat Heroes</header>;
};

export default Header;
