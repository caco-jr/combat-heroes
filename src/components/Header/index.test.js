import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import Header from './index';

describe('With Enzyme', () => {
  it('Header shows "Combat Heroes"', () => {
    const app = shallow(<Header />);

    expect(app.find('header').text()).toEqual('Combat Heroes');
  });
});

describe('With Snapshot Testing', () => {
  it('Header shows "Combat Heroes"', () => {
    const component = renderer.create(<Header />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
