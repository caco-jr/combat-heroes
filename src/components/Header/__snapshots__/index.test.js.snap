// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing Header shows "Combat Heroes" 1`] = `
<header
  className="container"
>
  Combat Heroes
</header>
`;
